### ISSUE TITLE
Circumscribed lesion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10081]
**Preferred term label:**
Circumscribed lesion

**Synonyms**
Often used to describe a mammography result

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1268666

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mucinous breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
it would probably be best not to limit the meaning of this term to breast masses (as implied by the synonym). Instead, what about
def: Applied to a lesion that is confined to a limited area, well defined and distinct from the surrounding tissue.

**Comment by MickeySegal** [comment]
I don't know how widely this terminology is used outside of mammography and breast ultrasound.  Here are the 2 instances that we are trying to represent:

https://www.ncbi.nlm.nih.gov/books/NBK538334/
"On mammography, circumscribed and lobulated breast lesions are common findings for a mucinous carcinoma. A circumscribed margin would favor the diagnosis of a pure mucinous carcinoma."

https://www.ncbi.nlm.nih.gov/books/NBK535345/
"Mammographic features of fibroadenomas are variable from a well-circumscribed discrete oval mass hypodense or isodense of breast glandular tissue to a mass with macro lobulation or partially obscured margins....  On []ultrasound, a fibroadenoma is typically seen as a well-circumscribed, round to ovoid, or macrolobulated mass with generally uniform hypoechogenicity".

But if you think circumscribed is used much more widely I'm fine with going with your approach.

**Comment by pnrobinson** [comment]
We would use a modifier such as Circumscribed if the modifier can be applied to many other terms. If we want to have a breast-specific finding, then it is probably better to create a term such as circumscribed breast fibroadenoma. However, at the end of the day ontologies always represent simplifications of a complex reality, and it is not a good idea to try to capture every single aspect of the data. I am guessing that for now it is best to make a Modifier called "circumscribed" that could be applied to any tumor (or indeed to other types of lesion). If down the line we find that this is not enough detail, we can add another modifier or term.

**Comment by MickeySegal** [comment]
That is fine.  We use modifiers a lot because life without them would drive people nuts.  The example I often use is bilateral episodic leg weakness.  One doesn't want to specify each permutation.

Here, circumscribed seemed to be a specific use for mammography, but you have the wider picture and I'm glad to use the modifier approach.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001047
- label: Circumscribed
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution and appearance are well-defined and localized to a specific area.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10081
- synonyms:
